<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResumeParser</class>
 <widget class="QMainWindow" name="ResumeParser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(97, 228, 156);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pdf_browse_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);


</string>
    </property>
    <property name="text">
     <string>Browse File (.pdf)</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1041</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>KacstArt</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RESUME PARSING</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="path_lbl">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>841</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_pic">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="result_lbl">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>490</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose button for data you need from the resume </string>
    </property>
   </widget>
   <widget class="QPushButton" name="name_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(114, 159, 207)}

</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ph_no_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>600</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(114, 159, 207)}

</string>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mail_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>660</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(114, 159, 207)}

</string>
    </property>
    <property name="text">
     <string>mail Id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tech_skills_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>600</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(114, 159, 207)}

</string>
    </property>
    <property name="text">
     <string>Technical Skills</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edu_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(114, 159, 207)}

</string>
    </property>
    <property name="text">
     <string>Education Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="soft_skills_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>660</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(114, 159, 207)}

</string>
    </property>
    <property name="text">
     <string>Soft Skills</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="display_lbl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(247, 243, 243);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
